src/main/java/frc/robot/subsystems/CameraSubsystem.java

```diff
@@ -42,7 +42,6 @@ public CameraSubsystem() {
       new PhotonPoseEstimator(
         FieldConstants.kAprilTagFieldLayout,
         PoseStrategy.MULTI_TAG_PNP_ON_COPROCESSOR,
-        m_camera2,
```

Comment:
I'm surprised this compiles.

Reply:
Upon further review, this looks correct. The PhotonVision API was brittle before, and now it's brittle in a different way now. :wink: